z3/src/ast.rs

```diff
@@ -1035,6 +1035,24 @@ impl<'ctx> String<'ctx> {
         }))
     }
 
+    /// Retrieves the underlying string
+    ///
+    /// If this is not a constant String, return None. Note that `to_string()`
```

Comment:
Can you clarify that this is referring to the `ToString` trait's method, which uses the `std::fmt::Display` implementation? Otherwise users/readers might be wondering where the `z3::ast::String::to_string` method is defined.